src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs

```diff
@@ -255,14 +249,14 @@ private ConsoleColors GetLogLevelConsoleColors(LogLevel logLevel)
             }
         }
 
-        private string GetScopeInformation()
+        private void GetScopeInformation(StringBuilder builder)
         {
             var current = ConsoleLogScope.Current;
-            var output = new StringBuilder();
             string scopeLog = string.Empty;
+            var length = builder.Length;
             while (current != null)
             {
-                if (output.Length == 0)
+                if (length == builder.Length)
```

Comment:
No need to have this in the while loop since you execute it only on the first iteration. It could be outside